addon/get-locales.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+const matchKey = '/locales/(.+)/translations$';
+
+export default function getLocales() {
+  return Ember.keys(requirejs.entries).reduce((locales, module) => {
+    var match = module.match(matchKey);
+    if (match) {
+      locales.push(match[1]);
+    }
+    return locales;
```

Comment:
This should probably wrap the result in `Ember.A()`. I think that's the best practice for things that might end up in a Glimmer `each`.